src/_sass/_images.scss

```diff
@@ -0,0 +1,22 @@
+.full-screen {
+  min-height: 100vh;
+  min-width: 100vw;
+  height: auto;
+  width: auto;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: 0;
+  background-position: top center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-color: #fff;
```

Comment:
If we are having a background image covering the whole view, is this `background-color` needed?